src/google/protobuf/compiler/objectivec/objectivec_file.cc

```diff
@@ -37,6 +37,7 @@
 #include <google/protobuf/io/zero_copy_stream_impl.h>
 #include <google/protobuf/stubs/stl_util.h>
 #include <google/protobuf/stubs/strutil.h>
+#include <algorithm> // std::find()
```

Comment:
Use two spaces before "// trailing line comment" (or just remove the comment)

https://google.github.io/styleguide/cppguide.html#Implementation_Comments